app/src/main/res/values/strings.xml

```diff
@@ -26,6 +26,13 @@
   <string name="topic_overview_see_more">See More</string>
   <string name="topic_download_text">(%s MB)</string>
   <string name="topic_story_progress_percentage">%s\%%</string>
+  <string name="continue_lesson">Continue with your Lesson</string>
+  <string name="more">MORE</string>
+  <string name="loading_topics_list_text">Loading topics list…</string>
+  <string name="error_loading_topics_list_text">Error loading topics list</string>
```

Comment:
Are this & the above string used? I think I integrated them originally to have a pending state while the topic controller provided a pending value.

Reply:
Removed this unused strings